modules/ROOT/partials/configuration/icon_list.adoc

```diff
@@ -163,7 +163,7 @@
 |`+template+` |image:icons/template.svg[template.svg] |`+template.svg+`
 |`+temporary-placeholder+` |image:icons/temporary-placeholder.svg[temporary-placeholder.svg] |`+temporary-placeholder.svg+`
 |`+text-color+` |image:icons/text-color.svg[text-color.svg] |`+text-color.svg+`
-|`+toc+` |image:icons/toc.svg[toc.svg] |`+toc.svg+`
```

Comment:
This wasn't renamed, so this should be reverted: https://github.com/tinymce/tinymce/blob/5c80ea7a2371dacefb9153dd49acb1fdbb9fd1b6/modules/oxide-icons-default/src/svg/toc.svg

- [ ] Revert this change